coremltools/converters/mil/mil/ops/defs/control_flow.py

```diff
@@ -538,6 +538,10 @@ class make_list(Operation):
     Returns
     -------
     List[*]
+
+    Attributes
+    ----------
+    dtype: fp16, fp32, i32, bool
```

Comment:
We don't need a separate attribute section. Instead it should be:

- init_length: <i32> (Optional, Default=1)
- dynamic_length: <bool> (Optional, default=True)
- elem_shape: <K, T> (Required)
- dtype: const<str> (Optional, Default="fp32"): possible values: {"bool", "fp16", "fp32", "int32" }

where T = "string", "int32"